### PR TITLE
Inject telemetry script if telemetry enabled hosted on dagster.io

### PR DESCRIPTION
**Pull request by salazarm**
## Summary & Motivation

The script doesn't exist yet so this just 404s atm but we will be adding it soon.

## How I Tested These Changes

Built the app, loaded it, confirmed everything works and that we load the script:

image

### CODE REVIEW

File: js_modules/dagster-ui/packages/app-oss/src/extractInitializationData.ts [hunk 1]
```diff
@@ -30,5 +30,10 @@ export const extractInitializationData = (): {
       }
     }
   }
+  if (value.telemetryEnabled) {
+    const script = document.createElement('script');
```
Comment (petehunt) [inline js_modules/dagster-ui/packages/app-oss/src/extractInitializationData.ts:34]: should we set `defer` and `async`?

File: js_modules/dagster-ui/packages/app-oss/src/extractInitializationData.ts [hunk 2]
```diff
@@ -30,5 +30,10 @@ export const extractInitializationData = (): {
       }
     }
   }
+  if (value.telemetryEnabled) {
```
Comment (petehunt) [inline js_modules/dagster-ui/packages/app-oss/src/extractInitializationData.ts:33]: testing this with telemetry on and off is important

Comment (salazarm) [reply js_modules/dagster-ui/packages/app-oss/src/extractInitializationData.ts:33]: yeah this path has been tested for the existing telemetry stuff but I can double check

**Review by bengotow — APPROVED** [review]
Nice! This looks good to me -- 

It looks like our OSS CSP allows `script-src 'nonce-NONCE-PLACEHOLDER' 'unsafe-inline' https: http: 'strict-dynamic'; `, so this will load? I didn't realize it allowed https: and http:

**Pull request merged**